puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_virt_who_configure.yaml

```diff
@@ -1,5 +1,28 @@
 - project:
     name: foreman_virt_who_configure
     defaults: plugin
+    dbs: 'postgresql'
+    parameters:
+      - string:
+          name: branch
+          default: master
+          description: "Branch to run test suite on"
+      - pr_parameters
+      - string:
+          name: foreman_branch
+          default: develop
+          description: "Foreman branch to run test suite against"
+      - string:
+          name: foreman_pr_git_url
+          default:
+          description: 'Foreman it URL to PR to test, e.g. git://github.com/user/foo.git'
+      - string:
+          name: foreman_pr_git_ref
+          default:
+          description: 'Foreman Git ref for PR to test, usually the branch name'
     jobs:
       - 'test_plugin_{name}_{branch}'
+    builders:
+      - clone_foreman
+      - test_katello
+      - test_foreman_virt_who_configure
```

Comment:
so far it does not modify Katello code directly (and I want to avoid it) but it requires Katello to live (e.g. organization#label) and introduces new SSO method that is only activated for [Katello::Api::Rhsm::CandlepinProxiesController](https://github.com/theforeman/foreman_virt_who_configure/blob/master/app/services/sso/basic_with_hidden.rb#L4)

I think it would be nice to know that this plugin does not break Katello. But if it's acceptable to add more code (e.g. monitoring incoming reports) to Katello code base and only use it if this plugin is installed (or make this plugin Katello dependency), maybe it's not worth it.